lib/serverspec/type/linux_audit_system.rb

```diff
@@ -0,0 +1,33 @@
+module Serverspec::Type
+  class LinuxAuditSystem < Base
+    def initialize(name=nil)
+      @name = 'linux_audit_system'
+      @runner = Specinfra::Runner
+      @rules_content = nil
+    end
+
+    def enabled?
+      status_of('enabled') == '1'
+    end
+
+    def running?
+      pid = status_of('pid')
+      (!pid.nil? && pid.size > 0 && pid != '0')
+    end
+
+    private
+
+    def rules
+      if @rules_content.nil?
+        @rules_content = @runner.run_command('/sbin/auditctl -l').stdout
+#.split("\n").map(&:chomp)
+      end
+      @rules_content || []
+    end
+
+    def status_of(part)
+      cmd = "/sbin/auditctl -s | grep \"^#{part}\" | awk '{ print $2 }'"
```

Comment:
ok, on my fedora 21 it's v2.4.3. this makes the status_of method a bit trickier, but not much. I'm going to have a look at this.